src/controllers/beneficial.owner.individual.controller.ts

```diff
@@ -1,11 +1,31 @@
-import { Request, Response } from "express";
+import { NextFunction, Request, Response } from "express";
+import { getApplicationData, prepareData, setApplicationData } from "../utils/application.data";
+import { ApplicationData, ApplicationDataType, beneficialOwnerIndividualType } from "../model";
 
 import { logger } from "../utils/logger";
 import * as config from "../config";
 
 export const get = (req: Request, res: Response) => {
-  logger.info(`GET THE INDIVIDUAL BENEFICIAL OWNER PAGE`);
+  logger.info(`GET ${config.BENEFICIAL_OWNER_INDIVIDUAL_PAGE}`);
+
+  const appData: ApplicationData = getApplicationData(req.session);
+
   return res.render(config.BENEFICIAL_OWNER_INDIVIDUAL_PAGE, {
-    backLinkUrl: config.LANDING_URL
+    backLinkUrl: config.LANDING_URL,
+    ...appData.beneficialOwnerIndividual
   });
 };
+
+export const post = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debug(`POST ${config.BENEFICIAL_OWNER_INDIVIDUAL_PAGE}`);
+
+    const data: ApplicationDataType = prepareData(req.body, beneficialOwnerIndividualType.BeneficialOwnerIndividualKeys);
+    setApplicationData(req.session, data, beneficialOwnerIndividualType.BeneficialOwnerIndividualKey);
```

Comment:
i think you need to build any sub-objects (like address) separately and add them to the data - see entityContoller - https://github.com/companieshouse/overseas-entities-web/blob/main/src/controllers/entity.controller.ts#L29